hcx-onboard/src/main/java/org/swasth/hcx/services/OnboardService.java

```diff
@@ -204,7 +204,9 @@ private void onboardProcess(HttpHeaders headers, OnboardRequest request, Map<Str
             Map<String, Object> onboardValidations = getConfig(request.getVerifierCode(), ONBOARD_VALIDATION_PROPERTIES);
             updateConfig(participantCode, onboardValidations);
         }
+        ArrayList<String> channel = new ArrayList<>(Arrays.asList(EMAIL,MOBILE));
         participant.put(USER_ID, userId);
+        participant.put("channel",channel);
```

Comment:
Instead of creating channel variable, Arrays.asList(EMAIL,MOBILE) can be directly provided here.